### PR TITLE
Sort document table

### PR DESCRIPTION
**Pull request by viktoriia-kosenko**

### CODE REVIEW

File: client/src/components/Dashboard.js [hunk 1]
```diff
@@ -63,10 +67,31 @@ class Dashboard extends Component {
       });
   };
 
+  sortOnClick = (sortKey, isSorted) => {
+    const documentsToSort = this.state.documents;
+    console.log(documentsToSort);
```
Comment (llh1) [inline client/src/components/Dashboard.js]: Remove console.log

File: client/src/components/Dashboard.js [hunk 2]
```diff
@@ -63,10 +67,31 @@ class Dashboard extends Component {
...
+    const sortedDocuments = sortDocuments(documentsToSort, sortKey);
+    console.log(sortedDocuments);
```
Comment (llh1) [inline client/src/components/Dashboard.js]: Remove console.log :)

File: client/src/components/Dashboard.js [hunk 3]
```diff
@@ -63,10 +67,31 @@ class Dashboard extends Component {
       });
   };
 
+  sortOnClick = (sortKey, isSorted) => {
```
Comment (llh1) [inline client/src/components/Dashboard.js]: Can you make sure that only one column can have a "sort up"/"sort down" icon? Like if you sort by document first, and then by due date, the sort icon of document should be back to "sort". So visually, we know which column is used to sort the table :)

**Review by llh1 — CHANGES_REQUESTED** [review]
Great job 👍  

Can you also try to keep the current sorting when clicking on the action button? For example, currently if I sort by from language, then I click on a Pick Document button or cancel button, the sorting change. But it should keep the rows sorted by from language.

File: client/src/components/Dashboard.js (hunk 1 above)
Comment (viktoriia-kosenko) [reply client/src/components/Dashboard.js]: Done

File: client/src/components/Dashboard.js (hunk 2 above)
Comment (viktoriia-kosenko) [reply client/src/components/Dashboard.js]: Done

File: client/src/components/Dashboard.js (hunk 3 above)
Comment (viktoriia-kosenko) [reply client/src/components/Dashboard.js]: Done

File: client/src/components/HeaderCell.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { Table, Icon } from "semantic-ui-react";
+
+function HeaderCell({ handleHeaderCellClick, headerCell, id, sortIconName }) {
+  const { header, sortKey } = headerCell;
+  return (
+    <Table.HeaderCell
+      onClick={() => handleHeaderCellClick(id, sortKey)}
+      id={id}
+    >
```
Comment (llh1) [inline client/src/components/HeaderCell.js:11]: You can also change the cursor so the user knows they can click on the column name, if you use in css something like this: `cursor: pointer`.

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/HeaderCell.js (hunk 4 above)
Comment (viktoriia-kosenko) [reply client/src/components/HeaderCell.js:11]: Done

**Review by llh1 — APPROVED** [review]

**Pull request merged**